api/envoy/extensions/transport_sockets/http_11_proxy/v3/upstream_http_11_connect.proto

```diff
@@ -35,5 +35,5 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //
 message Http11ProxyUpstreamTransport {
   // The underlying transport socket being wrapped.
```

Comment:
Suggest adding a comment here that says something like "If not set, plaintext is used."

Reply:
Gotcha, will do.